### ISSUE TITLE
h1-6 are not sizes (accessibility issue)

### ISSUE THREAD

**Issue by chasm** [smooth-code/smooth-ui#101]
https://smooth-ui.smooth-code.com/docs-components-typography

Under headings: "Six size of headings are available."

The 1-6 in `h1`-`h6` are not sizes. They have nothing whatsoever to do with how the text is styled in the browser. That is the remit of CSS.

1-6 have to do with *nesting*. The top-level heading *must* be `h1`. Subsections of that article would use `h2`. If a subsection is further divided, those sub-subsections would use `h3`. Unless one is writing an academic paper, `h4` to `h6` will almost never be used.

This is an accessibility issue. For more than two decades people in the accessibility community have been patiently explaining that these headings are not about size, with little success. As smooth-ui appears to care very much about accessibility, perhaps you could update this documentation to reflect the true purpose of the `h1` to `h6` elements. All sizing should be by CSS, not by choice of HTML element. (An exception is `small`, which can be used to mean "fine print".)

**Comment by chasm** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

**Comment by neoziro** [comment]
Yeah you are right, it could be renamed. I was inspired by Bootstrap to create this component. As you can see, [they have the same issue](https://getbootstrap.com/docs/4.2/content/typography/#headings) but I think we could make it better in Smooth UI. Open to propositions for a renaming!

**Comment by jacekschae** [comment]
How about just removing - _size of_

The sentence would read:
Six headings are available.

**Comment by chasm** [comment]
I like @jacekschae's idea, but even better would be "A top-level heading (h1) and five subsection headings (h2-h6) are available. Please use these in accordance with [current best practices](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). They are not intended to control font size: use CSS for that when necessary."

**Comment by neoziro** [comment]
@chasm OK. Can you update documentation with it?
